WordPress/metadata/release_notes.txt

```diff
@@ -1,10 +1,10 @@
-- Unsaved changes preview for draft posts
-- Feature image settings in Latest Post block
-- HTML/Visual mode switch notices
-- Full Embed block preview
-- Locale Picker visuals
-- Longer uploads with standalone VideoPress
-- Up-to-date Atomic site previews
-- Media Picker talkback media types
-- Non-Latin filename uploads
-- Clear feature image in post settings
+* [*] Pages List: Added "Copy Link" button to context menu [https://github.com/wordpress-mobile/WordPress-Android/pull/16321]
+* [*] Editor: Improved editor performance by reducing memory usage [https://github.com/wordpress-mobile/WordPress-Android/pull/16490]
+* [*] Block Editor: Prevent non-functional '+' icon displaying in toolbar on self-hosted sites [https://github.com/wordpress-mobile/WordPress-Android/pull/16507]
```

Comment:
Kept this line in the WordPress draft release notes only, and removed it from the Jetpack ones, as this change is related to self-hosted sites. \cc @leandroalonso

Reply:
@AliSoftware, yes, looks great to me! You're right that this doesn't apply to Jetpack-connected sites.